.github/scripts/findUnusedKeys.sh

```diff
@@ -6,7 +6,7 @@ LIB_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd ../../ && pwd)"
 
```

Comment:
do you know what is the purpose of this file?

Reply:
What I understand its a script used in gh action for removing unused styles